### PR TITLE
Use Tape's t.plan.

### PR DESCRIPTION
**Pull request by mponizil**
Not sure if this was an old Tape API or just legacy from QUnit, but it looks like Tape@2.3.2 prefers `t.plan`.

### CODE REVIEW

**Comment by mponizil** [comment]
Hmm, I'm befuddled with what to do about `t.end`. When it's there, a test may pass without fulfilling the plan, but without it, tests don't fail, they just hang.

https://github.com/substack/tape/issues/25 - either they fixed this or something similar?
https://github.com/substack/tape/issues/81 - someone else wondering the same

File: test/view.js [hunk 1]
```diff
@@ -107,12 +111,16 @@ test('Null template option is discarded.', 1, function(t) {
   t.end();
 });
 
-test('Undefined options does not throw.', 0, function(t) {
-  var view = new Quilt.View;
+test('Undefined options does not throw.', function(t) {
+  t.plan(1);
+  t.doesNotThrow(function() {
+    new Quilt.View;
+  });
```
Comment (braddunbar) [inline test/view.js:118]: :clap:  Good stuff.

**Comment by braddunbar** [comment]
Thanks @mponizil!  I think `t.end()` is required because otherwise a failure looks like an asynchronous test failure (hence the hang).  Either way, adding all the plans is great.  :smiley:

**Comment by mponizil** [comment]
Hmm, but check out: https://github.com/pathable/quilt/blob/master/test/view.js#L54

`t.end()` is intentionally omitted because when it's there (and you revert `continue` back to `return`), it will skip instead of fail. According to Tape docs, [`t.end()` will](https://github.com/substack/tape/blob/master/readme.markdown#tend):

> Declare the end of a test explicitly.

Sounds like it should still fail when plan isn't fulfilled- but that's not what I observe (`skips` run w/ `return` bug, `passing` run w/ `continue`):

``` diff
--- skips   2014-04-24 15:37:03.000000000 -0400
+++ passing 2014-04-24 15:38:00.000000000 -0400
@@ -37,28 +37,29 @@
 ok 20 should be equal
 # Other data attributes are ignored.
 ok 21 (unnamed assert)
+ok 22 (unnamed assert)

...

-1..30
-# tests 30
-# pass  30
+1..31
+# tests 31
+# pass  31

 # ok
```

**Comment by braddunbar** [comment]
Hmm...very strange.  I guess we should be explicit about it in the meantime.  Updated in 88a8221!

**Comment by mponizil** [comment]
Heh, that was more intended as an argument for omission. If you revert `continue` back to `return`, using `t.end()` suggests the test suite passes. It doesn't actually pass, it just runs 30 tests instead of 31.

Perhaps there's a better way to write that test- but in the mean time I'd argue for hanging over a false negative.

**Comment by braddunbar** [comment]
Ah!  I misunderstood.  You're right though, it doesn't fail based on the test plan.

However, it looks like updating `tape` fixes the issue (the test fails appropriately).  Updated in e52dc52.

**Comment by mponizil** [comment]
Much better!

**Pull request merged**